src/Akeneo/Tool/Bundle/ApiBundle/EventSubscriber/CheckRequestContentSizeSubscriber.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Tool\Bundle\ApiBundle\EventSubscriber;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * This subscriber will watch the API auth route and throw a 413 if the request content size is too large.
```

Comment:
```suggestion
 * This subscriber will watch an auth route and throw a 413 if the request content size is too large.
```
Just in case we use it to protect other routes